lib/utils/checkConfigExists/index.spec.js

```diff
@@ -0,0 +1,27 @@
+const {checkConfigExists} = require('./index.js');
```

Comment:
There should be spaces surrounding the require like so:
```
const { checkConfigExists } = require('./index');
```

Reply:
Let's add linting per default as a pre commit hook then :)